Content.Client/VendingMachines/UI/VendingMachineMenu.xaml

```diff
@@ -1,6 +1,26 @@
-﻿<DefaultWindow xmlns="https://spacestation14.io">
-    <ItemList Name="VendingContents"
-              SizeFlagsStretchRatio="8"
-              VerticalExpand="True">
-    </ItemList>
+﻿<DefaultWindow xmlns="https://spacestation14.io"
+    MinSize="650 250"
+    SetSize="675 250">
+    <BoxContainer Orientation="Horizontal">
+        <!-- List of vendable items -->
+        <ItemList Name="VendingContents" MinWidth="350" SizeFlagsStretchRatio="8" HorizontalExpand="True" VerticalExpand="True" />
+
+        <Control MinSize="10 10" />
+
+        <BoxContainer Orientation="Vertical" MinWidth="275" MaxWidth="275" SizeFlagsStretchRatio="2" HorizontalExpand="True" VerticalExpand="True">
+            <Button Disabled="True">
```

Comment:
I went back and realised I forget to check if this is used, please don't hack buttons like this.